src/seedu/addressbook/ui/TextUi.java

```diff
@@ -165,5 +166,13 @@ private static String getIndexedListForViewing(List<String> listItems) {
     private static String getIndexedListItem(int visibleIndex, String listItem) {
         return String.format(MESSAGE_INDEXED_LIST_ITEM, visibleIndex, listItem);
     }
+    
+    String getPrintableString(Printable... printables) {
+    	String printableStr = "";
+    	for (Printable printable : printables) {
```

Comment:
Good job using foreach loops :+1:  but you still have indentation problems in your code. Our coding standard requires you to use 4 spaces instead of tabs. You can configure Eclipse to convert tabs to spaces automatically.